src/gdb_packet.c

```diff
@@ -167,18 +176,19 @@ gdb_packet_s *gdb_packet_receive(void)
 {
 	packet_state_e state = PACKET_IDLE; /* State of the packet capture */
 	uint8_t rx_checksum = 0;
+	gdb_packet_s *packet_buffer = gdb_full_packet_buffer();
```

Comment:
To prevent confusion between this and the global, perhaps rename the local to `packet`?

Reply:
Sure. During development I actually used the names `_packet_buffer` and `packet_buffer`. I'll change the name to what you suggest.